pkg/services/secrets/kvstore/sql.go

```diff
@@ -223,3 +223,50 @@ func (kv *secretsKVStoreSQL) Rename(ctx context.Context, orgId int64, namespace
 		return err
 	})
 }
+
+// GetAll this returns all the secrets stored in the database. This is not part of the kvstore interface as we
+// only need it for migration from sql to plugin at this moment
+func (kv *secretsKVStoreSQL) GetAll(ctx context.Context) ([]Item, error) {
+	var items []Item
+	err := kv.sqlStore.WithDbSession(ctx, func(dbSession *sqlstore.DBSession) error {
+		return dbSession.Find(&items)
+	})
+	if err != nil {
+		kv.log.Debug("error getting all the items", "err", err)
+		return nil, err
+	}
+
+	// decrypting value
+	kv.decryptionCache.Lock()
+	defer kv.decryptionCache.Unlock()
+	for i := range items {
+		var decryptedValue []byte
+		if cache, ok := kv.decryptionCache.cache[items[i].Id]; ok && items[i].Updated.Equal(cache.updated) {
+			kv.log.Debug("got secret value from decryption cache", "orgId", items[i].OrgId, "type", items[i].Type, "namespace", items[i].Namespace)
+			items[i].Value = cache.value
+			continue
+		}
+
+		decodedValue, err := b64.DecodeString(items[i].Value)
+		if err != nil {
+			kv.log.Debug("error decoding secret value", "orgId", items[i].OrgId, "type", items[i].Type, "namespace", items[i].Namespace, "err", err)
```

Comment:
Yes, those were copied from the other methods. I think we can try to change them on all the methods. @guicaulada What do you think?

Reply:
I think since this is such a small change it would be ok to change all of the Debug logs that should be Error logs to Error in a single commit without impacting much of the review or affecting the scope of the PR.

Reply:
Fixed here 736928b3379725e6ba2d799f2cb4d5cdc3d10142